src/app/app.module.ts

```diff
@@ -3,14 +3,20 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HomePageModule } from './modules/home-page/home-page.module';
+import { PointsShopPageModule } from './modules/pointsshop-page/pointsshop-page.module';
+import { SharedModule } from './modules/shared/shared.module';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+	SharedModule,
+	HomePageModule,
+	PointsShopPageModule
```

Comment:
There are bad tabs, aren't there? :D
```suggestion
    AppRoutingModule,
    SharedModule,
    HomePageModule,
    PointsShopPageModule
```